state/networking/fffd.bat.tpl

```diff
@@ -1,7 +1,21 @@
 # THIS FILE IS CONTROLLED BY SALTSTACK!
 #
 # BATMAN Interface
+{% if grains['id'].startswith('kvm') -%}
```

Comment:
Maybe use something like `'kvm' in grains['roles']` and add an according role to `/etc/salt/minion.d/local.conf`?